### ISSUE TITLE
FunctionOnlyReturningConstant should ignore actual functions

### ISSUE THREAD

**Issue by sczerwinski** [detekt/detekt#3388]
## Expected Behavior of the rule
By default, `actual` functions in multiplatform projects should be ignored when evaluating `FunctionOnlyReturningConstant`.

There should be a configuration option `ignoreActualFunction`, which would allow changing this behaviour.

#### commonMain
```kotlin
expect class Foo {
    fun foo(param: Any): Boolean
}
```
#### jvmMain
```kotlin
actual class Foo {
    actual fun foo(param: Any): Boolean = checkSomeCondition(param)
}
```
#### jsMain
```kotlin
actual class Foo {
    actual fun foo(param: Any): Boolean = false    // <-- FunctionOnlyReturningConstant should be ignored
}
```

## Context
Similar to `ignoreOverridableFunction`.

Implementing an `actual` function, is conceptually similar to overriding a function from an interface/abstract class.
It makes sense for such a function to provide complex logic on some platforms, and only return a constant value on others.

**Comment by cortinico** [comment]
> There should be a configuration option `ignoreActualFunction`, which would allow changing this behaviour.

That's a valid addition. Do you want to add this extra config option @sczerwinski ?

**Comment by sczerwinski** [comment]
@cortinico, I'll try to prepare a pull request, but it might take a few days for me to get familiar with the codebase.
